docs/install_packages_and_deploy_cluster.md

```diff
@@ -0,0 +1,95 @@
+
+# How to install the packages and deploy local cluster on Linux
+
+To deploy and run a cluster on your Linux development machine, install the runtime and common SDK generated as the build output. 
+
+## Prerequisites
+
+The following operating system versions are supported for development:
+
+* Ubuntu 16.04 (`Xenial Xerus`)
+
+Ensure you have run the build using the -createinstaller option to generate the debian files
+```
+./runbuild.sh -createinstaller
+```
+
+## Installation steps
+
+### Update your APT sources
+To install the SDK and the associated runtime package via the apt-get command-line tool, you must first update your Advanced Packaging Tool (APT) sources.
+
+1. Open a terminal.
+
+2. Add the `dotnet` repo to your sources list.
+
+    ```bash
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ xenial main" > /etc/apt/sources.list.d/dotnetdev.list'
+    ```
+
+3. Add the new Gnu Privacy Guard (GnuPG, or GPG) key to your APT keyring.
+
+    ```bash
+    sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 417A0893
+    ```
+
+4. Add the official Docker GPG key to your APT keyring.
+
+    ```bash
+    sudo apt-get install curl
+    sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    ```
+
+5. Set up the Docker repository.
+
+    ```bash
+    sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+    ```
+
+6. Refresh your package lists based on the newly added repositories.
+
+    ```bash
+    sudo apt-get update
+    ```
+
+### Install and set up the Service Fabric Debians for local cluster setup
+
+After you have updated your sources, you can install the runtime and SDK, confirm the installation, and agree to the license agreement. Please cd to the location in the output directory where the debians are placed i.e. repo.root/out/build.prod/FabricDrop/deb/. Install the runtime and sdk Debian packages named as servicefabric_x.x.x.x.deb and servicefabric_sdkcommon_y.y.y.deb, where x.x.x.x and y.y.y is the actual version numbers in the same of the file. Please replace the commands below with the actual files names in the folder 
+
+```bash
+sudo dpkg -i servicefabric_x.x.x.x.deb
+sudo apt-get install -fy
+sudo dpkg -i servicefabric_sdkcommon_x.x.x.deb
+sudo apt-get install -fy
```

Comment:
It's confusing that when I run this command it says:
```
$ sudo apt-get install -fy
Reading package lists... Done
Building dependency tree
Reading state information... Done
Correcting dependencies... Done
The following packages will be REMOVED:
  servicefabricsdkcommon
0 upgraded, 0 newly installed, 1 to remove and 23 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
(Reading database ... 59795 files and directories currently installed.)
Removing servicefabricsdkcommon (1.2.0) ...
In PostRm - remove
```
It removed the package? It didn't install it?

Reply:
Looks like this hapens because the previous servicefabric installation failed..

Reply:
Looks like I missed an earlier line... I forgot that Azure installs docker differently, and I had to add the above repository for the dependency to work.  Kind of weird.

Reply:
Anyway, all of this part is fine.  Mistake is mine.